pkg/utils/flow/flow.go

```diff
@@ -205,15 +205,15 @@ func (e *execution) runNode(ctx context.Context, id TaskID) {
 		log := e.log.WithField(logKeyTask, id)
 
 		start := time.Now().UTC()
-		log.Debugf("Started at %s", start)
+		log.Info("Started")
```

Comment:
Can we leave it with `debug` to prevent pollution of the log?

Reply:
For me it made things a lot clearer since then I could see when a task started, not just when it finished. I'd argue it's a trade-off that is OK but I went ahead and removed it.